### PR TITLE
genders: fix `python3` reference.

### PR DESCRIPTION
**Pull request by miles170**
See #108008, and also update the license.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by miles170** [comment]
The build works fine on Ubuntu22.04 x86_64. 
I have no idea why the arm build failed. I will try to reproduce it on the MacBook M1.

**Comment by miles170** [comment]
> The build works fine on Ubuntu22.04 x86_64. 
> I have no idea why the arm build failed. I will try to reproduce it on the MacBook M1.

works fine on MacBook Air M1 (Monterey)

```
HOMEBREW_VERSION: 3.6.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6e2b162c4786e075323f038d46bfb566d91889e7
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: e40092b8f33064f4443ef4efe04f4ab9581b90c6
Core tap last commit: 16 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: vim
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.1.6 build 1316
Git: 2.37.3 => /opt/homebrew/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.5.1-arm64
CLT: 13.4.0.0.1.1651278267
Xcode: N/A
Rosetta 2: false
```

**Comment by carlocab** [comment]
Still failed. Try doing `make` and `make install` in separate steps:
```ruby
system "make"
system "make", "install"
```

File: Formula/genders.rb [hunk 1]
```diff
@@ -35,8 +35,10 @@ class Genders < Formula
   end
 
   def install
-    ENV["PYTHON"] = which("python3")
+    ENV["PYTHON"] = which("python3.10")
     system "./configure", "--prefix=#{prefix}", "--with-java-extensions=no"
+    # Run in separate steps to avoid install failure.
+    system "make"
```
Comment (carlocab) [inline Formula/genders.rb:41]: Still failed. Let's try this.
```suggestion
    ENV.deparallelize
```

**Comment by miles170** [comment]
Close this pr since genders depends_on python formula which can be any python3.X version

**Pull request closed without merging**